extensions/credentials/runtime/src/main/java/io/quarkus/credentials/CredentialsProvider.java

```diff
@@ -2,11 +2,21 @@
 
 import java.util.Map;
 
+/**
+ * Service Programming Interface that provides an indirection between credentials consumers (e.g. agroal)
+ * and implementers (e.g. vault). Detailed information is available in th dedicated guide.
```

Comment:
Small typo in `th` but may be this doc can be simplified a bit, may be just `Provides an indirection between credentials consumers such as Agroal and implementers such as Vault.`...

Reply:
done. thanks.